src/sentry/integrations/bitbucket/webhook.py

```diff
@@ -70,8 +70,12 @@ def __call__(self, organization, event):
         except Repository.DoesNotExist:
             raise Http404()
 
-        if repo.config.get('name') != event['repository']['full_name']:
-            repo.config['name'] = event['repository']['full_name']
+        # update our data if repo name has changed
+        repo_full_name = event['repository']['full_name']
+        if repo.config.get('name') != repo_full_name:
+            repo.config['name'] = repo_full_name
+            repo.name = repo_full_name
+            repo.url = u'https://bitbucket.org/{}'.format(repo_full_name)
```

Comment:
@lobsterkatie it would be good to do something similar to what we're doing with github then. I'm pretty sure bitbucket sends the url as well. :)

Reply:
So, BB has `website`, which it describes as "The URL to the repository's website when the code is hosted for a specific website." Not sure how to parse this, and have not had a chance to set up a test BB repo to play with. Do you know if that's the right data/do you have a setup you can test it with easily?

See: https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html#EventPayloads-entity_repository

Reply:
You could use `repository.links.html.href`. It looks like it should have the web link to the repository as well. The `website` key might be the user entered site that the repo is *for*. Like sentry.io instead of bitbucket.org/getsentry/sentry

Reply:
Oh, so _that's_ what they meant by that confusing sentence! Thanks for figuring that out, @markstory!

Meanwhile, unfortunately, I don't think `repository.links.html.href` will get us what we're looking for, at least judging by this:
![image](https://user-images.githubusercontent.com/14812505/52675137-af482580-2eda-11e9-9e35-118941a2ce01.png)
(this is what you turn out to get if you click on "Repository property" underneath the table I linked above)

It seems like all of the urls in `links` are API urls, not the public url of the repo (which is what we're going for here). That said - we actually don't have a "BitBucket Enterprise/BitBucket Server/BitBucket On-Prem" integration the way we do with GitHub. Are we likely to? If not, can we leave it the way it is and not worry about it?

Reply:
That is so bizarre that they would put an API URL in `links.html`, but the docs show it 😢

Reply:
I put it back to being `"bitbucket.com/whatever"` for the moment.